docs/mobile/streamers.mdx

```diff
@@ -22,4 +23,22 @@ submenu:
       height: 80
       width: 90
     videos: []
+  - imgPlaceholder: 'notJust.dev (Vadim Savin)'
+    titleCard: 'React Native, Expo.dev, Typescript, JavaScript, etc.'
+    links:
+      - type: 'X'
+        url: 'https://x.com/VadimNotJustDev/'
+      - type: 'Web'
+        url: 'https://www.notjust.dev/'
+      - type: 'Ig'
+        url: 'https://www.instagram.com/VadimNotJustDev/'
+      - type: 'YouTube'
+        url: 'https://www.youtube.com/channel/UCYSa_YLoJokZAwHhlwJntIA'
+    cover:
+      src: '/streamers/vadim-headshot.webp'
+      height: 80
+      width: 90
+    createdAt: '2024-01-30T00:00:00Z'
```

Comment:
Estas fechas se quitaron, podrás eliminarlas? createdAt y updatedAt

Reply:
Of cors! Perdón por la demorísima!